### ISSUE TITLE
All data should be preserved by saving collections in .json files. [1.5Xpts]

### ISSUE THREAD

**Issue by Cisco-the-wayword** [jlvFlores/catalog-of-my-things#7]
All the data pre-user-inputs should be stored in appropriate, well-labelled .json files for easy accessibility
